_episodes/13-looking-up-data.md

```diff
@@ -49,7 +49,7 @@ The next exercise demonstrates this two stage process in full.
 >* Give the column a name e.g. "Journal-Details"
 >* In the expression box you need to write some GREL where the output of the expression is a URL which can be used to retrieve data (the format of the data could be HTML, XML, JSON, or some other text format)
 >
->In this case we are going to use the CrossRef API: [https://github.com/CrossRef/rest-api-doc)](https://github.com/CrossRef/rest-api-doc). Read more about the CrossRef service: [http://www.crossref.org](http://www.crossref.org). Note that API providers may impose rate limits or have other requirements for using their data, so it's important to check the site's documentation. To comply with API rate limits, use the Throttle Delay setting to specify the number of milliseconds between URL requests. CrossRef, for instance, [asks users](https://github.com/CrossRef/rest-api-doc#etiquette) to "specify a User-Agent header that properly identifies your script or tool and that provides a means of contacting you via email using 'mailto:'." User-agent headers provide administrators with user information that facilitates better administration and moderation of the API, and it is generally good etiquette to include a header with any API request.
+>In this case we are going to use the CrossRef API: [https://api.crossref.org/](https://api.crossref.org/). Read more about the CrossRef service: [https://crossref.org](https://crossref.org). Note that API providers may impose rate limits or have other requirements for using their data, so it's important to check the site's documentation. To comply with API rate limits, use the Throttle Delay setting to specify the number of milliseconds between URL requests. CrossRef, for instance, [asks users](https://gitlab.com/crossref/rest_api/-/blob/main/resources/swagger/description.md#etiquette) to "specify a User-Agent header that properly identifies your script or tool and that provides a means of contacting you via email using 'mailto:'." User-agent headers provide administrators with user information that facilitates better administration and moderation of the API, and it is generally good etiquette to include a header with any API request.
```

Comment:
I think it would be better to reference the etiquette section on the published documentation page  https://api.crossref.org/swagger-ui rather than the gitlab repo?

Reply:
It has no `id`/anchor there, though:

![Screenshot 2022-11-03 at 13 14 38](https://user-images.githubusercontent.com/9948149/199717814-a2c27f18-362d-48f5-87d8-fb7e91dda9d3.jpg)

Reply:
I think that's their problem rather than ours - it's not great from the learners perspective to be linked to the gitlab page I don't think? Perhaps we could raise an issue with them to add anchors to that file?

Reply:
@jas58 what do you think?

Reply:
Let me clarify the text for the issue to post on their repo:
Hello and thank you for hosting this documentation.  We would like to link to your etiquette section and noticed there isn't an HTML anchor in the public swagger site.  Our learners are early beginners, so I would prefer to link to the swagger site and avoid linking to the github repo.  Are subdivision anchors for href's possible in the near future?

Reply:
That tips link suggested looks good to me - I agree that would be good to use as the link

Reply:
@katrinleinweber This works for us.  If you agree, please submit revision. Thank you.

Reply:
Happy New Year, @katrinleinweber ! We are trying to wrap up the old year's PRs and wanted to follow up with you on this.  Do you have any further questions for us?

Reply:
No further questions. If you want to use a different link URL, I'm perfectly fine with that. That's why I've enabled the `Allow edits … by maintainers` option on this MR ;-)